src/components/App/App.component.jsx

```diff
@@ -9,6 +9,7 @@ import SecretPage from '../../pages/Secret';
 import Private from '../Private';
 import Fortune from '../Fortune';
 import Layout from '../Layout';
+import StyledHeader from '../Header/Header.component';
```

Comment:
1. Should use `Header` instead of `StyledHeader`, keep the component name the same with filename

2. For readability, in `./components/Header/` folder, we could create file `index.js` with content:
 ```js
 export { default } from './Header.component';
 ```
By doing this 
```js
import StyledHeader from '../Header/Header.component';
```
becomes
 
```js
import StyledHeader from '../Header';
```

Similarly for `import` in other files